net-news/sfeed/sfeed-1.6.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 should work here.

Reply:
IIRC 8 was missing `savedconfig` (maybe that got changed or I'm misremembering ?).